### ISSUE TITLE
Arch build breaking in pandoc invocation

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#781]
megamind6155 reports on the AUR that notcurses is breaking, and helpfully provides a build log:

I'm not able to build the package. I'm getting a build failure. Here is a yay build log:
https://bin.snopyta.org/?badaefaa5e7597e3#5F5QxQJPEqNoHA2BjvKByiHY7yYbbZwvK5YNMpzDND52

```
:: (1/1) Parsing SRCINFO: notcurses
==> Making package: notcurses 1.6.0-1 (Friday 10 July 2020 02:52:23 PM)
==> Retrieving sources...
  -> Found v1.6.0.tar.gz
==> Validating source files with sha256sums...
    v1.6.0.tar.gz ... Passed
==> Making package: notcurses 1.6.0-1 (Friday 10 July 2020 02:52:24 PM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found v1.6.0.tar.gz
==> Validating source files with sha256sums...
    v1.6.0.tar.gz ... Passed
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Extracting v1.6.0.tar.gz with bsdtar
==> Starting prepare()...
-- The C compiler identification is GNU 10.1.0
-- The CXX compiler identification is GNU 10.1.0
-- Check for working C compiler: /sbin/cc
-- Check for working C compiler: /sbin/cc - works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /sbin/c++
-- Check for working CXX compiler: /sbin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Requested multimedia engine: ffmpeg
-- Requested build mode: RelWithDebInfo
-- Found PkgConfig: /sbin/pkg-config (found version "1.7.3")
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Checking for one of the modules 'tinfo>=6.1;ncursesw>=6.1'
-- Checking for module 'libavcodec>=57.0'
--   Found libavcodec, version 58.54.100
-- Checking for module 'libavformat>=57.0'
--   Found libavformat, version 58.29.100
-- Checking for module 'libavutil>=56.0'
--   Found libavutil, version 56.31.100
-- Checking for module 'libswscale>=5.0'
--   Found libswscale, version 5.5.100
-- Looking for qrcodegen/qrcodegen.h
-- Looking for qrcodegen/qrcodegen.h - found
-- Found Python3: /sbin/python3.8 (found version "3.8.3") found components: Development Interpreter
-- Configuring done
-- Generating done
-- Build files have been written to: /home/me/.cache/yay/notcurses/src/notcurses-1.6.0/build
==> Sources are ready.
==> Making package: notcurses 1.6.0-1 (Friday 10 July 2020 02:52:29 PM)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Removing existing $pkgdir/ directory...
==> Starting build()...
Scanning dependencies of target notcurses_lines.3.man
Scanning dependencies of target notcurses
[  0%] Building man page notcurses_lines.3
[  0%] Building C object CMakeFiles/notcurses.dir/src/lib/blit.c.o
pandoc: /home/me/.cache/yay/notcurses/src/notcurses-1.6.0/doc/man/man3/notcurses_lines.3.md: openBinaryFile: does not exist (No such file or directory)
make[2]: *** [CMakeFiles/notcurses_lines.3.man.dir/build.make:81: notcurses_lines.3] Error 1
make[2]: *** Deleting file 'notcurses_lines.3'
make[1]: *** [CMakeFiles/Makefile2:532: CMakeFiles/notcurses_lines.3.man.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[  1%] Building C object CMakeFiles/notcurses.dir/src/lib/debug.c.o
[  1%] Building CXX object CMakeFiles/notcurses.dir/src/lib/direct.cpp.o
[  1%] Building C object CMakeFiles/notcurses.dir/src/lib/fade.c.o
[  2%] Building C object CMakeFiles/notcurses.dir/src/lib/fd.c.o
[  2%] Building CXX object CMakeFiles/notcurses.dir/src/lib/ffmpeg.cpp.o
[  3%] Building C object CMakeFiles/notcurses.dir/src/lib/fill.c.o
[  3%] Building C object CMakeFiles/notcurses.dir/src/lib/input.c.o
[  3%] Building C object CMakeFiles/notcurses.dir/src/lib/menu.c.o
[  4%] Building C object CMakeFiles/notcurses.dir/src/lib/metric.c.o
[  4%] Building C object CMakeFiles/notcurses.dir/src/lib/notcurses.c.o
[  5%] Building CXX object CMakeFiles/notcurses.dir/src/lib/oiio.cpp.o
[  5%] Building CXX object CMakeFiles/notcurses.dir/src/lib/plot.cpp.o
[  5%] Building C object CMakeFiles/notcurses.dir/src/lib/reader.c.o
[  6%] Building C object CMakeFiles/notcurses.dir/src/lib/reel.c.o
[  6%] Building C object CMakeFiles/notcurses.dir/src/lib/render.c.o
[  6%] Building C object CMakeFiles/notcurses.dir/src/lib/selector.c.o
[  7%] Building C object CMakeFiles/notcurses.dir/src/lib/terminfo.c.o
[  7%] Building CXX object CMakeFiles/notcurses.dir/src/lib/visual.cpp.o
[  8%] Linking CXX shared library libnotcurses.so
[  8%] Built target notcurses
make: *** [Makefile:161: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
error making: %!s(func() string=0x6406b5c68ed0)
```

this definitely looks like some pandoc weirdness. Targeted for 1.6.1.

**Comment by dankamongmen** [comment]
It looks like he's doing a cached build with yay, and it's possibly missing a file.

**Comment by ask6155** [comment]
It's me who made the post in the AUR. I removed the cache and tried to build again. Same error. I checked in the cache folder and the file does exist!

**Comment by ask6155** [comment]
I also tried:
`pandoc -t pdf .cache/yay/notcurses/src/notcurses-1.6.0/doc/man/man3/notcurses_lines.3.md`
it gave `openBinaryFile: does not exist`

**Comment by ask6155** [comment]
I think it is an issue with firejail (I run firejail) and that is not letting pandoc find the markdown files. Better report it there.

**Comment by dankamongmen** [comment]
> I think it is an issue with firejail (I run firejail) and that is not letting pandoc find the markdown files. Better report it there.

ok interesting, i've never even heard of that. i'm going to leave this open until i hear more from you -- thanks for verifying that it was something more disturbing than an invalid cache. i appreciate the report and would like to get to the bottom of this. there might be a way for me to work around this behavior if i look into firejail....

**Comment by dankamongmen** [comment]
I see https://aur.archlinux.org/packages/cheat/?O=10&PP=10#comment-736806. I also see that a recent release of firejail included an application profile for pandoc: https://firejail.wordpress.com/download-2/release-notes/

**Comment by dankamongmen** [comment]
Unless I hear differently, I'm assuming this was indeed due to firejail. Please let me know if that's not the case.

**Comment by ask6155** [comment]
Ok that issue is fixed, but still my build is failing, this time, I think it is a compile test this time:
```
[  0%] Built target notcurses_lines.3.man
[  8%] Built target notcurses
[  9%] Built target notcurses-view.1.man
[  9%] Built target notcurses_input.3.man
[ 10%] Built target notcurses_fade.3.html5
[ 10%] Built target notcurses_error.3.man
[ 10%] Built target notcurses_fds.3.html5
[ 11%] Built target notcurses_cell.3.html5
[ 12%] Built target notcurses-pydemo.1.man
[ 12%] Built target notcurses_cell.3.man
[ 12%] Built target notcurses.3.html5
[ 12%] Built target notcurses_directmode.3.html5
[ 12%] Built target notcurses_stats.3.html5
[ 13%] Built target notcurses-input.1.man
[ 13%] Built target notcurses_init.3.man
[ 14%] Built target notcurses_input.3.html5
[ 14%] Built target notcurses_plane.3.man
[ 15%] Built target notcurses_directmode.3.man
[ 15%] Built target notcurses-tester.1.html5
[ 16%] Built target notcurses_lines.3.html5
[ 17%] Built target notcurses_refresh.3.html5
[ 26%] Built target notcurses-demo
[ 27%] Built target notcurses_selector.3.html5
[ 31%] Built target notcurses++-static
[ 31%] Built target ncneofetch.1.html5
[ 31%] Built target notcurses_stdplane.3.html5
[ 32%] Built target notcurses-view.1.html5
[ 32%] Built target notcurses-ncreel.1.html5
[ 32%] Built target notcurses_channels.3.html5
[ 32%] Built target notcurses_fade.3.man
[ 32%] Built target notcurses.3.man
[ 33%] Built target notcurses_render.3.html5
[ 33%] Built target notcurses_metric.3.html5
[ 41%] Built target notcurses-static
[ 42%] Built target notcurses_output.3.html5
[ 42%] Built target ncneofetch.1.man
[ 43%] Built target notcurses_palette.3.man
[ 44%] Built target notcurses_menu.3.man
[ 44%] Built target notcurses_plane.3.html5
[ 45%] Built target notcurses_fds.3.man
[ 45%] Built target notcurses-input.1.html5
[ 45%] Built target notcurses_init.3.html5
[ 46%] Built target notcurses_reel.3.man
[ 46%] Built target notcurses_menu.3.html5
[ 46%] Built target notcurses_multiselector.3.man
[ 47%] Built target notcurses_multiselector.3.html5
[ 47%] Built target notcurses_palette.3.html5
[ 48%] Built target ncneofetch
[ 48%] Built target notcurses_plot.3.man
[ 49%] Built target notcurses-tetris.1.html5
[ 49%] Built target notcurses-tetris.1.man
[ 50%] Built target notcurses_plot.3.html5
[ 50%] Built target notcurses_output.3.man
[ 51%] Built target notcurses_reader.3.html5
[ 51%] Built target notcurses_reel.3.html5
[ 51%] Built target notcurses_refresh.3.man
[ 51%] Built target notcurses_error.3.html5
[ 51%] Built target notcurses_render.3.man
[ 51%] Built target notcurses_selector.3.man
[ 52%] Built target notcurses_stats.3.man
[ 52%] Built target notcurses_stop.3.man
[ 53%] Built target notcurses_stop.3.html5
[ 54%] Built target notcurses_visual.3.man
[ 54%] Built target notcurses_metric.3.man
[ 54%] Built target notcurses_visual.3.html5
[ 54%] Built target notcurses-demo.1.man
[ 55%] Built target notcurses-demo.1.html5
[ 55%] Built target notcurses-ncreel.1.man
[ 55%] Built target notcurses_reader.3.man
[ 55%] Built target notcurses-pydemo.1.html5
[ 56%] Built target notcurses-tester.1.man
[ 57%] Built target notcurses_channels.3.man
[ 57%] Built target notcurses_stdplane.3.man
[ 62%] Built target notcurses++
[ 63%] Built target widestomp
[ 74%] Built target notcurses-tester
[ 75%] Built target wcwidth
[ 76%] Built target qrcode
[ 77%] Built target unidamage
[ 78%] Built target visual
[ 79%] Built target sgr
[ 80%] Built target rotator
[ 80%] Built target rgb
[ 81%] Built target blocks
[ 82%] Built target rgbbg
[ 83%] Built target geom
[ 84%] Built target notcurses-view
[ 84%] Built target selector
[ 85%] Built target ncpp_build_exceptions
[ 86%] Built target resize
[ 87%] Built target fileroller
[ 87%] Built target blitters
[ 88%] Built target dirlines
[ 89%] Built target ncpp_build
[ 90%] Built target cjkscroll
[ 90%] Built target notcurses-ncreel
[ 91%] Built target linuxconjammer
[ 92%] Built target linuxconsole
[ 92%] Built target procroller
[ 92%] Built target mathtext
[ 93%] Built target pixels
[ 94%] Built target zalgo
[ 95%] Built target scroll
[ 95%] Built target direct
[ 96%] Built target multiselect
[ 96%] Built target firstline
[ 97%] Built target reader
[ 97%] Built target notcurses-tetris
[ 97%] Built target vizdirect
[ 98%] Built target menu
[ 99%] Built target notcurses-input
[100%] Built target dirgb
Install the project...
-- Install configuration: "RelWithDebInfo"
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/notcurses/direct.h
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/notcurses/ncerrs.h
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/notcurses/nckeys.h
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/notcurses/notcurses.h
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Cell.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/CellStyle.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Direct.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/FDPlane.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Menu.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/MultiSelector.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/NCAlign.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/NCBox.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/NCKey.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/NCLogLevel.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/NotCurses.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Palette256.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Plane.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Plot.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Reader.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Reel.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Root.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Selector.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Subproc.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Tablet.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/TabletCallback.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Utilities.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/Visual.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/_exceptions.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/_flag_enum_operator_helpers.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/_helpers.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/ncpp.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/include/ncpp/internal/Helpers.hh
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/pkgconfig/notcurses++.pc
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/cmake/notcurses/notcursesConfig.cmake
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/cmake/notcurses/notcursesConfigVersion.cmake
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/pkgconfig/notcurses.pc
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/PurpleDrank.jpg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/aidsrobots.jpeg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/changes.jpg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli01.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli02.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli03.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli04.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli05.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli06.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli07.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli08.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli09.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli1.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli10.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli11.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli12.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli13.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli14.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli15.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli16.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli17.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli18.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli19.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli2.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli20.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli21.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli22.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli23.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli24.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli25.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli26.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli27.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli28.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli29.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli3.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli30.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli31.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli32.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli33.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli34.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli35.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli36.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli37.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli38.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli39.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli4.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli40.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli41.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli42.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli43.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli44.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli45.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli5.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli6.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli7.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/chunli8.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/covid19.jpg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/dsscaw-purp.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/eagles.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/fm6.mkv
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/lamepatents.jpg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/megaman2.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/normal.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/notcursesI.avi
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/samoa.avi
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/tetris-background.jpg
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/warmech.bmp
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/notcurses/worldmap.png
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/ncneofetch.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-demo.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-input.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-ncreel.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-pydemo.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-tester.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-tetris.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man1/notcurses-view.1
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_cell.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_channels.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_directmode.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_error.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_fade.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_fds.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_init.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_input.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_lines.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_menu.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_metric.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_multiselector.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_output.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_palette.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_plane.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_plot.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_reader.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_reel.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_refresh.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_render.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_selector.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_stats.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_stdplane.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_stop.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/man/man3/notcurses_visual.3
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/doc/notcurses/NEWS.md
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/doc/notcurses/OTHERS.md
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/doc/notcurses/README.md
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/doc/notcurses/TERMS.md
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/share/doc/notcurses/USAGE.md
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-demo
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-demo" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-input
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-input" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-ncreel
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-ncreel" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/ncneofetch
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/ncneofetch" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-tester
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-tester" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-tetris
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-tetris" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-view
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/bin/notcurses-view" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses.so.1.6.6
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses.so.1
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses.so.1.6.6" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses.so
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses++.so.1.6.6
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses++.so.1
-- Set runtime path of "/home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses++.so.1.6.6" to ""
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses++.so
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses.a
-- Installing: /home/me/.cache/yay/notcurses/pkg/notcurses/usr/lib/libnotcurses++.a
running install
running build
running build_py
creating build
creating build/lib.linux-x86_64-3.8
creating build/lib.linux-x86_64-3.8/notcurses
copying src/notcurses/__init__.py -> build/lib.linux-x86_64-3.8/notcurses
copying src/notcurses/build_notcurses.py -> build/lib.linux-x86_64-3.8/notcurses
copying src/notcurses/notcurses.py -> build/lib.linux-x86_64-3.8/notcurses
running build_ext
generating cffi module 'build/temp.linux-x86_64-3.8/_notcurses.c'
creating build/temp.linux-x86_64-3.8
building '_notcurses' extension
creating build/temp.linux-x86_64-3.8/build
creating build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -fno-semantic-interposition -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=x86-64 -mtune=generic -O3 -pipe -fno-plt -march=native -O2 -pipe -fstack-protector-strong -fno-plt -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/python3.8 -c build/temp.linux-x86_64-3.8/_notcurses.c -o build/temp.linux-x86_64-3.8/build/temp.linux-x86_64-3.8/_notcurses.o
build/temp.linux-x86_64-3.8/_notcurses.c:541:14: fatal error: notcurses/direct.h: No such file or directory
  541 |     #include <notcurses/direct.h>
      |              ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
error: command 'gcc' failed with exit status 1
==> ERROR: A failure occurred in package().
    Aborting...
error making: %!s(func() string=0x5aa6930c8e80)

```

**Comment by dankamongmen** [comment]
I'll create a new bug on this issue. I think I know what's up.

**Comment by dankamongmen** [comment]
@ask6155 , your issue ought be fixed in 1.6.6-2, just uploaded to the AUR. See #813 for more info.
